brainiak/utils/utils.py

```diff
@@ -105,6 +105,71 @@ def fast_inv(a):
         raise
 
 
+def sumexp_stable(data):
```

Comment:
why do we need this instead of a softmax function directly? are we going to use this besides the calculation in softmax?